Makefile

```diff
@@ -604,6 +605,27 @@ $(SBOM): $(ROOTFS_TAR) | $(INSTALLER)
 	rm -rf $(TMP_ROOTDIR)
 	$(QUIET): $@: Succeeded
 
+generate_metadata: $(ROOTFS_TAR) | $(INSTALLER)
+	$(QUIET): $@: Begin
+	$(eval TMP_ROOTDIR := $(shell mktemp -d))
+	# this is a bit of a hack, but we need to extract the rootfs tar to a directory, and it fails if
+	# we try to extract character devices, block devices or pipes, so we just exclude the dir.
+	# when syft supports reading straight from a tar archive with duplicate entries,
+	# this all can go away, and we can read the rootfs.tar
+	# see https://github.com/anchore/syft/issues/1400
+	tar xf $< -C $(TMP_ROOTDIR) --exclude "dev/*"
+	bash tools/get-alpine-pkg-source.sh -v -s /tmp/sources/alpine -e $(TMP_ROOTDIR) 
+	bash tools/get-kernel-source.sh -v -s /tmp/sources/kernel 
+	echo $(shell $(ROOTDIR))
+	for i in $$(find . -name 'go.sum'); \
+	do								 		   \
+		go-sources-and-licenses sources -d $$(dirname $$i) --recursive --out /tmp/sources/go; \
```

Comment:
Something I don't understand. The tool `go-sources-and-licenses` is installed inside the build container, but it is called from the `Makefile`? Can I run `make <whatever>` like I can call `make sbom`?

Reply:
Question remains.

Reply:
go install is not needed in the Dockerfile. I have moved it to the makefile under the build-tools/bin folder i.e the temp GOPATH. The file is getting downloaded and working as expected. Removed the install from the Dockerfile.